commands/permissions.go

```diff
@@ -36,12 +35,13 @@ var RemovePermissionsCmd = &cobra.Command{
 }
 
 var ShowRoleCmd = &cobra.Command{
```

Comment:
this command has been moved, so we can delete it from this file

Reply:
I thought we wanted this to continue working using the old `permissions show [role_name]` command, but with a deprecation warning.

Reply:
You're completely right, I oversaw the deprecation property, sorry :facepalm: